app.py

```diff
@@ -10,13 +10,12 @@
 from datetime import datetime
 import json
 import hashlib
-import manage_docker
-import manage_rancher
+from rancher1_handler import rancher1_handler
```

Comment:
Shouldn't manage_docker and manage_rancher be deleted then?

Reply:
They *are* deleted!

Reply:
The two files @sychan